### PR TITLE
Check if account is a multisign

### LINKED ISSUE #383: Not checking if account is a multisign kill node
> [log] 2017-01-09 14:23:54 | POST /api/multisignatures/sign from 127.0.0.1
> [FTL] 2017-01-09 14:23:54 | System error - {"message":"Cannot read property 'indexOf' of null","stack":"TypeError: Cannot read property 'indexOf' of null\n    at Immediate.<anonymous> (/Users/vsat/git/lisk/lisk-node/modules/multisignatures.js:371:33)\n    at runCallback (timers.js:641:20)\n    at tryOnImmediate (timers.js:610:5)\n    at processImmediate [as _immediateCallback] (timers.js:582:5)"}
> [inf] 2017-01-09 14:23:54 | Cleaning up...
> [inf] 2017-01-09 14:23:54 | Cleaned up successfully
> [inf] 2017-01-09 14:23:54 | caught internal exit

was using old version of lisk, but the bug remain:

https://github.com/LiskHQ/lisk/blob/development/modules/multisignatures.js#L360

scope.sender.multisignatures is not checked if exists. So trying to sign as multi over a legit account which is not a multiaccount is killing the node

### PR DESCRIPTION
**Pull request by MaciejBaj**
Handle situation when sender has no multisignatures field.

closes #383

### CODE REVIEW

File: test/api/multisignatures.js [hunk 1]
```diff
@@ -61,7 +61,10 @@ function confirmTransaction (transactionId, passphrases, done) {
 				secret: passphrase,
 				transactionId: transactionId
 			}, function (err, res) {
-				node.expect(res.body).to.have.property('success').to.be.ok;
+				var possibleErr = err || res.body.err;
```
Comment (4miners) [inline test/api/multisignatures.js]: Looks like `possibleErr` is not used.

Comment (MaciejBaj) [inline test/api/multisignatures.js]: sure, I will remove it

File: test/api/multisignatures.js [hunk 2]
```diff
@@ -547,3 +549,60 @@ describe('POST /api/multisignatures/sign (transaction)', function () {
...
+		node.get('/api/multisignatures/pending?publicKey=' + accounts[0].publicKey, function (err, res) {
+			node.expect(res.body).to.have.property('success');
+			node.expect(res.body).to.have.property('success').to.be.ok;
+			node.expect(res.body).to.have.property('transactions').that.is.an('array');
+			node.expect(res.body.transactions.length).to.equal(0);
+			done();
+		});
+	});
+
+	after(function (done) {
```
Comment (karmacoma) [inline test/api/multisignatures.js]: @MaciejBaj I don't think we need to revert transaction here.

Comment (MaciejBaj) [inline test/api/multisignatures.js]: sure

**Comment by karmacoma** [comment]
Closing in favor of #405 on named branch.

**Pull request closed without merging**